base_images/debian_base-setup.sh

```diff
@@ -16,6 +16,10 @@ REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+install_automation_tooling
```

Comment:
@edsantiago just FYI that I'm attempting this.

It's currently not working because some basic deps (like `git`) are missing for the installer to run.  Should be relatively easy to fix.